subsys/usb/usbc/Kconfig

```diff
@@ -0,0 +1,36 @@
+# USBC stack configuration options
+
+# Copyright (c) 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Workaround for not being able to have commas in macro arguments
+DT_COMPAT_USBC_PORT := usbc-port
+
+menuconfig USBC_STACK
+	bool "USBC Stack Support"
+	default $(dt_compat_enabled,$(DT_COMPAT_USBC_PORT))
```

Comment:
OK, now I see this is being used like a "real" compatible, but then the comment in the binding's description that it's "common fields" doesn't make sense. Something is wrong here. I would have expected individual driver IPs to have their own compatibles in their own bindings. Each of those could `select` the right Kconfig symbol to turn on the common infrastructure in addition to doing anything IP-specific in their own drive rfiles. Unless there is some sort of common IP for USB type C, the way 16550 is for serial?

Reply:
I updated the comment in usbc-port.yaml